tests/src/test/scala/ch/renku/acceptancetests/workflows/FlightsTutorial.scala

```diff
@@ -29,18 +29,34 @@ trait FlightsTutorial extends GivenWhenThen with Matchers with ScalatestMatchers
     And("enables git lfs")
     console %> c"git lfs install --local"
 
-    And("imports a dataset from the dataverse")
-    console %> c"renku dataset import 10.7910/DVN/WTZS4K".userInput("y")
+    And("installs virtualenv")
+    console %> c"python3 -m pip install virtualenv"
+
+    And("creates virtualenv directory")
+    console %> c"virtualenv venv"
+
 
     And("adds some Python packages to the requirements.txt")
     console %> (c"echo pandas==0.25.3" >> "requirements.txt")
     console %> (c"echo seaborn==0.9.0" >> "requirements.txt")
+    console %> (c"echo renku==0.10.4" >> "requirements.txt")
+    console %> (c"echo papermill<1.2.0" >> "requirements.txt")
+    console %> (c"echo requests>=2.20.0" >> "requirements.txt")
+    console %> (c"echo jupyterhub==1.1.0" >> "requirements.txt")
+    console %> (c"echo nbresuse==0.3.3" >> "requirements.txt")
+    console %> (c"echo jupyterlab-git==0.20.0" >> "requirements.txt")
+    console %> (c"echo renku==0.10.4" >> "requirements.txt")
+
 
     And("installs the packages")
-    console %> c"pip install -r requirements.txt"
+    console %> c"./venv/bin/pip install -r requirements.txt"
```

Comment:
why not activate the environment instead of specifying the full path into the venv directory? You have to then deactivate it at the end or in the case of an error.

Reply:
The tests were hanging when I tried to activate the the environment. However, it might be possible to set the same environment variables that the activate script does to avoid specifying the full path.